common/shlibs

```diff
@@ -3874,8 +3874,9 @@ libnvpair.so.3 zfs-2.0.3_2
 libgmio.so gmio-0.4.1_1
 libjsonnet.so.0 jsonnet-0.14.0_2
 libjsonnet++.so.0 jsonnet-0.14.0_2
-libigdgmm.so.12 intel-gmmlib-22.0.1_1
+libigdgmm.so.11 intel-gmmlib-21.3.1_1
 libigfxcmrt.so.7 intel-media-driver-21.3.5_1
+libigdgmm.so.11 intel-gmmlib-19.4.1_1
```

Comment:
This reintroduces a duplicate line in "common/shlibs"; I don't know if this is desired but I'd like to point this out